tests/testthat/test_functions.R

```diff
@@ -0,0 +1,25 @@
+context("Tests of package functions")
+
+mock_data <- data.frame(
+  va = c(1:10),
+  vb = c(2:11),
+  vc = factor(c(rep(1:2, 5)), labels = c("one", "two")),
+  vd = c(rep(c("jeden", "dwa", "x", "y", "z"), 2)),
```

Comment:
I can see some Polish words here as well. Please change them to English ones.

Reply:
fixed